### ISSUE TITLE
'oead.byml.Hash' object has no attribute 'pop'

### ISSUE THREAD

**Issue by cievic** [NiceneNerd/BCML#326]
After updating bcml thru a pop up in its launcher it started refusing to launch which I ended up solving with a fresh install of bcml.

but after I made a fresh install and continued with adding a mod it started giving me the error 'oead.byml.Hash' object has no attribute 'pop' any time I did something from installing a mod, updating a mod and remerging mods.

In detail it states:
```
Traceback (most recent call last):
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\_api.py", line 550, in remerge
    install.refresh_merges()
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\install.py", line 661, in refresh_merges
    merger.perform_merge()
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\util.py", line 462, in timed_function
    res = func(*args, **kwargs)
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\mergers\quests.py", line 167, in perform_merge
    add.pop("prev_quest", None)
AttributeError: 'oead.byml.Hash' object has no attribute 'pop'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\_api.py", line 33, in status_run
    data = func(*args, **kwargs)
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\install.py", line 324, in do_and_refresh
    res = func(*args, **kwargs)
  File "c:\users\vic\desktop\python\lib\site-packages\bcml\_api.py", line 558, in remerge
    raise Exception(
Exception: There was an error merging your mods. 'oead.byml.Hash' object has no attribute 'pop'
Note that this could leave your game in an unplayable state.
```
A few posts back I read a very similar error to my one but it said the fix is to downgrade which if that is the fix I just need to know how I downgrade my version of bcml. Other fixes if there are would be useful too.

**Comment by GingerAvalanche** [comment]
This (and a few other bugs) is already fixed in the github code, we just need to make another release.

You can follow the instructions on the main page to install bcml from source (which will get you those fixes) or you can use `pip install bcml==3.8.4` to roll back

**Comment by GingerAvalanche** [comment]
Fixed in v3.8.6
